### DISCUSSION TITLE
source code link in hvm is incorrect

### DISCUSSION THREAD

**Discussion by stevenhuyn** [zicklag/blog#4] [General]
Hi thank you so much for your blog post on HVM: https://zicklag.github.io/blog/interaction-nets-combinators-calculus/

Just one part, the link that goes to `https://github.com/HigherOrderCO/Wikind/blob/master/IC/_.kind2` no longer exists. Could you replace it with a perma link instead?

**Comment by zicklag** [comment]
Hey there! Thanks for the shout out!

I've just deployed a fix, and here's the updated link: https://github.com/HigherOrderCO/Kindex/blob/fe9c7248782f99eb543567bcf2408004bc155f9d/IC/_.kind2#L1.

Note that this points to the closest version of the file that I know of at the time of writing, but there is a new version of the file ( without the explanatory comments, though ) [here](https://github.com/HigherOrderCO/Kindex/blob/master/Apps/IC/_.kind2). The latest explanatory comments have been moved to a new repo [here](https://github.com/VictorTaelin/Interaction-Calculus).
